salt/kubelet/kubelet.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Kubernetes Kubelet Server
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+After=docker.service
+Requires=docker.service
+
+[Service]
+WorkingDirectory=/var/lib/kubelet
+EnvironmentFile=-/etc/kubernetes/node_labels
+EnvironmentFile=-/etc/kubernetes/config
+EnvironmentFile=-/etc/kubernetes/kubelet
+ExecStart=/usr/bin/hyperkube kubelet \
+	    $KUBE_LOGTOSTDERR \
+	    $KUBE_LOG_LEVEL \
+	    $KUBELET_API_SERVER \
+	    $KUBELET_ADDRESS \
+	    $KUBELET_PORT \
+	    $KUBELET_HOSTNAME \
+	    $KUBE_ALLOW_PRIV \
+        $KUBELET_NODE_LABELS \
+	    $KUBELET_ARGS
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I understand this is only for testing this works, because is generally a bad idea to overwrite the `kubelet.service` from Salt...

If you want to add `$KUBELET_NODE_LABELS` to the service it would be better to add it in [the service file](https://build.opensuse.org/package/view_file/Virtualization:containers/kubernetes/kubelet.service?expand=1), then add `KUBELET_NODE_LABELS=` in `/etc/kubernetes/kubelet` using the Jinja template we have, and just watch that file (Salt will reload the service automatically, so you will not need that `service.systemctl_reload`)...